app/models/calculation_result.rb

```diff
@@ -107,8 +107,9 @@ def client_owns_main_home?
     capital_items(:properties).dig(:main_home, :value)&.positive?
   end
 
-  def client_main_home_rows
-    main_home_rows(prefix: "")
+  def main_home_data
```

Comment:
Good question! So for additional properties, the `client_` or `partner_` prefix indicates who _owns_ the additional property. Now, CFE acts as though the client's main home is owned by the client, but CCQ knows that sometimes the client's main home is actually owned by the partner, or even is jointly owned (we just _tell_ CFE it's owned by the client to keep things simple).

So while it is the client's main home, it's not necessarily the client-_owned_ main home, and as such I thought it would be worth dropping the prefix, which does carry those extra connotations.